neo/SmartContract/InteropService.cs

```diff
@@ -18,6 +18,9 @@ namespace Neo.SmartContract
 {
     public static partial class InteropService
     {
+        public const int MaxStorageKeySize = 512;
+        public const int MaxStorageValueSize = 65536;
```

Comment:
```suggestion
        public const int MaxStorageValueSize = ushort.MaxValue;
```

Reply:
`ushort.MaxValue` is `65535`, is not `65536`.

Reply:
I think that Is better to have the limit in two bytes